packages/components/src/pill/pill.js

```diff
@@ -0,0 +1,7 @@
+export function Pill( { children } ) {
```

Comment:
A new component is born 🎉  - can we also get a line added to the [changelog](https://github.com/woocommerce/woocommerce-admin/blob/main/packages/components/CHANGELOG.md) about this new component?

Reply:
Done in 0fe3e6d7638c8f8238869080f58591942ee203ab